drivers/mtd/w25qxxxjv.c

```diff
@@ -976,8 +1009,8 @@ static int w25qxxxjv_write_page(struct w25qxxxjv_dev_s *priv,
 
       if (ret < 0)
         {
-          ferr("ERROR: QSPI_MEMORY failed writing address=%06x\n",
-               address);
+          ferr("ERROR: QSPI_MEMORY failed writing address=%06"PRIx32"\n",
```

Comment:
have you seen https://stackoverflow.com/questions/586928/how-should-i-print-types-like-off-t-and-size-t ?

Reply:
Yes, I did read this thread, but it did not become clear to me what the correct way to print a `off_t` is.   Are you referring to:

> To print off_t:
>
> `printf("%jd\n", (intmax_t)x);`

Reply:
yes

Reply:
Thanks for your review.  I applied your improvement proposals.